FastSimulation/Tracking/plugins/TrajectorySeedProducer.cc

```diff
@@ -104,38 +105,34 @@ TrajectorySeedProducer::TrajectorySeedProducer(const edm::ParameterSet& conf):
 bool
 TrajectorySeedProducer::pass2HitsCuts(const TrajectorySeedHitCandidate & innerHit,const TrajectorySeedHitCandidate & outerHit) const
 {
-
-    const DetLayer * innerLayer = 
-	measurementTrackerEvent->measurementTracker().geometricSearchTracker()->detLayer(innerHit.hit()->det()->geographicalId());
-    const DetLayer * outerLayer = 
-	measurementTrackerEvent->measurementTracker().geometricSearchTracker()->detLayer(outerHit.hit()->det()->geographicalId());
+  const DetLayer * innerLayer =
+    measurementTrackerEvent->measurementTracker().geometricSearchTracker()->detLayer(innerHit.hit()->det()->geographicalId());
+  const DetLayer * outerLayer =
+    measurementTrackerEvent->measurementTracker().geometricSearchTracker()->detLayer(outerHit.hit()->det()->geographicalId());
+  vector<BaseTrackerRecHit const *> innerHits;
+  innerHits.push_back((BaseTrackerRecHit*&&)innerHit);
```

Comment:
Sorry, I don`t exactly remember why I did that !! I have now edited the code. Thanks.